src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.List;
+
+/**
+ * Represents a helper class to save and retrieve task list.
+ */
+public class Storage {
+
+    private static final String FILE_PATH = "./data/duke.txt";
+
+    private static final Set<String> VALID_TASK_TYPE = Set.of("T", "D", "E");
```

Comment:
Nice use of casing for constants! Can consider using the plural VALID_TASK_TYPES, since it is a set.
Same as VALID_DONE_STATUSES